### PR TITLE
feat(participant): remove in-app notification for participant VSCODE-662

### PR DESCRIPTION
**Pull request by gagik**
We decided to remove the notification for Copilot extension so we don't spam existing users.

This essentially reverts changes in https://github.com/mongodb-js/vscode/pull/875

### CODE REVIEW

File: src/telemetry/telemetryService.ts [hunk 1]
```diff
@@ -517,6 +517,7 @@ export default class TelemetryService {
       command,
       error_code: errorCode,
       error_name: errorName,
+      error_message: message,
```
Comment (alenakhineika) [inline src/telemetry/telemetryService.ts:520]: We can not track the message itself, because it can contain user's sensitive data.

Comment (gagik) [reply src/telemetry/telemetryService.ts]: oh sorry this was meant to go in the https://github.com/mongodb-js/vscode/pull/900 PR also... but glad that was caught, had no idea! I'm not too sure what we could to do to improve error tracking beyond this

Comment (alenakhineika) [reply src/telemetry/telemetryService.ts]: Yeah, this is the cause of the problem in the first place, that we can not track a message, and why Kevin was asking about maybe different specialized tools we could utilize. I'm also currently not sure what we could do to improve error tracking.

Comment (gagik) [reply src/telemetry/telemetryService.ts]: then I could revert and leave it as is for now; hopefully we'll at least see the reduction in the unknown error we see with exportToPlayground

Comment (alenakhineika) [reply src/telemetry/telemetryService.ts]: So let's exclude `error_message` for now and think about ways to improve error tracking later in a separate pr.

Comment (gagik) [reply src/telemetry/telemetryService.ts]: Yep, removed it here

File: src/telemetry/telemetryService.ts [hunk 2]
```diff
@@ -527,14 +526,4 @@ export default class TelemetryService {
   trackParticipantResponse(props: ParticipantResponseProperties): void {
     this.track(TelemetryEventTypes.PARTICIPANT_RESPONSE_GENERATED, props);
   }
-
-  trackCopilotIntroductionClicked(props: CopilotIntroductionProperties): void {
```
Comment (alenakhineika) [inline src/telemetry/telemetryService.ts:531]: You can also remove the `CopilotIntroductionProperties` type.

**Pull request merged**